mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
+  try {
+    const order = await Order.findByIdAndRemove(req.params.orderId);
+    if (!order) {
+      return res.status(404).json({ message: "Order does not exist" });
+    }
+    res.status(200).json({ message: "Order deleted successfully" });
+  } catch (err) {
```

Comment:
The `catch` block should be handled more robustly. Consider logging the error using a logging mechanism like Winston or Morgan to track and debug errors. Additionally, it would be better to return a 500 status code with a generic error message instead of exposing the internal error details to the client.